frontend/src/App.tsx

```diff
@@ -50,25 +48,32 @@ function Users() {
 }
 
 const operations = {
-  '+': (left: number, right: number): number => left + right,
-  '-': (left: number, right: number): number => left - right,
-  '*': (left: number, right: number): number => left * right,
-  '/': (left: number, right: number): number => left / right,
-}
+  "+": (left: number, right: number): number => left + right,
+  "-": (left: number, right: number): number => left - right,
+  "*": (left: number, right: number): number => left * right,
+  "/": (left: number, right: number): number => left / right,
+};
 type CalculatorProps = {
-  left: number
-  operator: keyof typeof operations
-  right: number
-}
-function Calculator({left, operator, right}: CalculatorProps) {
-  const result = operations[operator](left, right)
+  left: number;
+  operator: keyof typeof operations;
+  right: number;
+};
+function Calculator({ left, operator, right }: CalculatorProps) {
+  fetch("/stablecoins")
```

Comment:
@popoaq  this should work as a result of setting the proxy in `package.json` so the request would go to `localhost:8080`
https://www.freecodecamp.org/news/how-to-make-create-react-app-work-with-a-node-backend-api-7c5c48acb1b0/

But it's still giving me 404s against our ktor server. I just tried it with a vanilla Express server and can get responses back from FE
I think there's still something we need to tweak on the server side, potentially related to CORS, for this to work.

Reply:
Update: After spending hours on this, I went with the non-proxy route but adding `CORS` to the server. Now I can render the response in the UI